static/js/common/upload-addon.js

```diff
@@ -337,11 +337,13 @@
                       // Specific messages for unlisted addons.
                       var isSideload = $('#id_is_sideload').is(':checked') || $new_form.data('addon-is-sideload');
                       var automaticValidation = $('#create-addon').data('automatic-validation') || $new_form.data('automatic-validation');
+                      var passedValidation = (!v.signing_summary || (!v.signing_summary.trivial && !v.signing_summary.low &&
+                                                                     !v.signing_summary.medium && !v.signing_summary.high));
```

Comment:
Hm. I'm not sure how much I like that we're duplicating these checks in the front end and the back end. If something changes, they're likely to get out of sync. It would be nice if we could annotate the validation on the server side so we're only doing the checks once.